csun-metalab/nomi
# NAM-463 -- UI: "Students" without emails need to have a fix for student profiles

# Description
Added logic to check for users with no email; rewired the front-end vuex for the proper route.

# Testing
1. Visit a normal student and make sure the profile loads up with no console errors.
2. Ensure when spamming the back button, the previous issue of the wrong profile loading up does not persist.
3. Go to matthew.fritz, visit the 2nd course list item, visit Donald Salpers profile and ensure the profile loads up with no errors.
